fdbserver/ClusterController.actor.cpp

```diff
@@ -254,6 +257,17 @@ ACTOR Future<Void> clusterWatchDatabase(ClusterControllerData* cluster,
 			    .detail("ChangeID", dbInfo.id);
 			db->serverInfo->set(dbInfo);
 
+			if (SERVER_KNOBS->ENABLE_ENCRYPTION && !recoveredDisk) {
+				// EKP singleton recruitment waits for 'Master/Sequencer' recruitment, execute wait for
+				// 'recoveredDiskFiles' optimization once EKP recruitment is unblocked to avoid circular dependencies
+				// with StorageServer initialization
+
+				wait(recoveredDiskFiles);
+				TraceEvent("CCWDB_RecoveredDiskFiles", cluster->id).log();
```

Comment:
nit: I think `.log()` is not needed?
```suggestion
				TraceEvent("CCWDB_RecoveredDiskFiles", cluster->id);
```

Reply:
`.log()` ensures that the code line is flushed before the variable `goes out of scope`; it is useful with the `named` TraceEvent variables. 
No harm adding it here, however, otherwise too the code should work fine.